src/components/Tooltip/Tooltip.js

```diff
@@ -384,6 +398,37 @@ export default class Tooltip extends Component {
           'aria-owns': tooltipId,
         };
 
+    const finalIcon = IconCustomElement ? (
+      <IconCustomElement
+        name={iconName}
+        className={`${prefix}--tooltip__custom-icon`}
```

Comment:
Is `className` here absolutely necessary?

Reply:
No, I will remove it.